README.md

```diff
@@ -194,6 +217,12 @@ Now Puppet knows where to download the module from when you include it in your s
     # github "java",     "1.6.0"
     include java
 
+### Updating modules
+
+* To check for modules updates defined in your `Puppetfile`, run `bundle exec librarian-puppet outdated`. You'll probably need a Github API Token [see above](#github-api-token) to avoid Github's API rate limits.
```

Comment:
Good point. It's accessible via the system keychain like this:

`sudo security  find-generic-password -l "GitHub API Token" -w`

Perhaps we can set `GITHUB_API_TOKEN` before a run this way?